reference/checkouts.v3.yml

```diff
@@ -8798,6 +8898,31 @@ components:
           type: string
           description: Coupon codes have a 50-character limit.
       x-internal: false
+    ApplyFeesRequest:
+      title: Fees Post/Put Request
+      type: object
+      properties:
+        type:
+          type: string
+          description: the type of the fee
+          enum:
+            - custom_fee
+        name:
+          type: string
+          description: name of the fee. e.g "Route Package Protection Insurance"
+        display_name:
+          type: string
+          description: display name of the fee targeting customers/shoppers e.g "Package Protection Insurance"
+        cost:
+          type: number
+          description: cost of the fee. 
+        tax_class_id:
+          type: integer
+          description: the tax class id of the fee. 0 => global default tax class id
```

Comment:
0 means global default tax, we need to find the class id in the server side and apply it for tax

Reply:
decided to not include tax_class_id atm. we always use default tax class id for fees this milestone (support Route)